plugins/registry/handler.go

```diff
@@ -199,13 +211,26 @@ func (rh *registryHandler) relay(ctx context.Context, p registryHandlerParams) {
 	}
 
 	if req.Method == http.MethodPut && resourceType == "manifests" {
-		logger.Info("tag:", resourceID)
-		logger.Info("model ID:", matches[1])
-		// DIGEST AT dockerContentDigest := resp.Header.Get("Docker-Content-Digest")
-		// CREATION TIME createdAt := time.Now()
+		// TODO Call create tag endpoint in artifact-backend.
+
+		// Deploy model.The previous operations are idempotent so it should be
+		// safe to repeat them if we fail here.
+		//
+		// TODO in the future the registry will handle more than model images,
+		// so this operation won't always be necessary. A much better pattern
+		// is publishing the push operation success as an event and let the
+		// clients to consume and act upon it (artifact to register the tag
+		// creation time, model to deploy the image...).
+
+		// TODO call UserModel or OrganizationModel depending on username == namespace.
```

Comment:
@heiruwu @tonywang10101 it's a bit cumbersome to leak the user/org business domain here to call different endpoints 🙃 In VDP we discussed reducing the number of org/user duplicated endpoints, in most of them the underlying logic is the same. Not required for the immediate future but worth considering in the Model domain as well.